depz/depz.py

```diff
@@ -840,6 +839,28 @@ def add_command (self, c, priority=0):
       self.commands[priority] = []
     self.commands[priority].append(c)
 
+  def queue_command (self, name, requested_on_cmd_line, f, allow_default_actions, alt_name=None):
```

Comment:
Maybe something like add_all_repo_command() or something?  To really differentiate it from add_command().  That's a bit of a mouthful, but at the moment, the name doesn't express the difference.

Reply:
I've renamed it to add_command_per_repo(), which seems fitting.